src/vmm/src/vstate/vcpu/aarch64.rs

```diff
@@ -311,10 +311,11 @@ mod tests {
         );
         assert!(err.is_err());
         assert_eq!(
-            err.err().unwrap().to_string(),
-            "Error configuring the vcpu registers: Failed to set processor state register: Bad \
-             file descriptor (os error 9)"
-                .to_string()
+            err.unwrap_err(),
+            super::Error::ConfigureRegisters(ArchError::SetOneReg(
+                6931039826524241986,
```

Comment:
Nit: This number occurs a couple times, can a constant be used? (even if its just `const TEST_NUMBER: u64 = 6931039826524241986;`)

Reply:
Converted number to hex to be consistent with other tests. Also removed second appearance of this value, because it was in the duplication of the vcpu config test.